Formula/mold.rb

```diff
@@ -42,6 +47,8 @@ def install
     ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
     ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version <= 1200)
 
+    rewrite_shebang detected_python_shebang(use_python_from_path: true), "update-git-hash.py"
```

Comment:
This needs to be reported upstream, at least. It seems unnecessarily restrictive to require a `/usr/bin/python3`.

Reply:
https://github.com/rui314/mold/pull/637